app/src/main/java/com/sun/heartrate/ui/heartbeat/HeartbeatContract.kt

```diff
@@ -1,16 +1,14 @@
 package com.sun.heartrate.ui.heartbeat
 
-import android.hardware.camera2.CameraManager
-
 interface HeartbeatContract {
     interface View {
-        fun getCameraManager(): CameraManager
         fun closeCamera()
+        fun displayHeatRate(rateNumber: Int)
+        fun displayGuideline(guideline: Int)
     }
     
     interface Presenter {
-        fun openCamera()
-        fun closeCamera()
-        fun checkCamera(): Boolean
+        fun currentProgressBarValue(cameraBootTime:Long, measurementTime:Long):Int
```

Comment:
Method nên bắt đầu bằng động từ. Presenter cũng không nên biết đến khái niệm ProgressBar. Nên sửa tên là loadAbc hoặc getAbc gì đó. Ví dụ:
```suggestion
        fun loadCameraBootProgression(cameraBootTime : Long, measurementTime : Long) : Int
```